src/app/shared/shared.module.ts

```diff
@@ -11,6 +11,7 @@ import { ObjectFilterPipe } from 'app/pipes/object-filter.pipe';
 import { OrderByPipe } from 'app/pipes/order-by.pipe';
 import { PhaseFilterPipe } from 'app/pipes/phase-filter.pipe';
```

Comment:
Do we want to keep this import? Could this be used anywhere else or not? What code are these shared services shared by?

Reply:
If it's not used in this file, I'm fine with removing the reference, but keeping the phase-filter.pipe file.

Reply:
I looked into it and I don't think the phase filter is used elsewhere. I removed it, checked the functionality, ran the unit and functional tests again, and everything passed. I'll remove the references from this file.